addons/kafka/opsdefinitions/quota.yaml

```diff
@@ -56,7 +56,7 @@ spec:
         podSpec:
           containers:
             - name: kafka-topic
-              image: docker.io/bitnami/kafka:3.3.2-debian-11-r54
+              image: apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/kafka:3.3.2-debian-11-r54
```

Comment:
why not use template?

{{ .Values.images.registry | default "docker.io" }}/{{ .Values.images.kafka.repository }}:{{ default .Chart.AppVersion .Values.images.kafka.tag }}

Reply:
These three YAML documents are not defined within the templates directory, and therefore do not support template. Should I move them there and use template?

Reply:
Should use template, otherwise the image registry can not be modified, and it will still be hardcoded.